monkestation/code/modules/clothing/shoes/miscellaneous.dm

```diff
@@ -0,0 +1,15 @@
+/obj/item/clothing/shoes/yes_slip
+	name = "Yes-Slip Shoes"
+	desc = "A pair of very slippery sneakers that even slide out of your view."
+	icon_state = "yesslip"
+	var/next_slip
+
+/obj/item/clothing/shoes/yes_slip/step_action()
+	. = ..()
+	if(next_slip <= world.time)
+		var/mob/living/carbon/wearer = loc
+		to_chat(wearer, "<span class='clowntext'>[pick("You slip flat on your face!", "Your shoes send you flying!", "An invisible leg of the Honkmother trips you!")]</span>")
+		playsound(loc, 'sound/misc/slip.ogg', 25, 1, -1)
+		playsound(loc, 'sound/effects/laughtrack.ogg', 50, 1, -1)
+		wearer.Paralyze(2 SECONDS, TRUE, TRUE)
```

Comment:
don't handle slip code here, see `/turf/open/handle_slip()`

Reply:
So that's where it was.
I couldn't find any mob procs about slipping at all.